server/emits.py

```diff
@@ -98,7 +98,7 @@ def user_message_handler(sid, data):
                 data
             )
             gesture[sid] = None
-            print("Received move: " + str(move))
+
             if not move:
                 print("Failed to find the requested block.")
```

Comment:
If you're taking out prints should this also be removed?

Reply:
I'd say no, since it's basically a placeholder for the "request more information / clarification" story. It's also just useful to make sure that the server heard you and your request wasn't lost in transit.